unittests/CppInterOp/VariableReflectionTest.cpp

```diff
@@ -121,13 +121,17 @@ TEST(VariableReflectionTest, GetVariableOffset) {
   EXPECT_TRUE((bool) Cpp::GetVariableOffset(Decls[1])); // N
 
   EXPECT_EQ(Cpp::GetVariableOffset(datamembers[0]), 0);
+//FIXME: Unable to compile these on Windows due to type conversion
+#ifdef _WIN32
+
+#else
   EXPECT_EQ(Cpp::GetVariableOffset(datamembers[1]),
-          ((unsigned long) &(c.b)) - ((unsigned long) &(c.a)));
+          (reinterpret_cast<unsigned long>(&(c.b)) - reinterpret_cast<unsigned long>(&(c.a))));
   EXPECT_EQ(Cpp::GetVariableOffset(datamembers[2]),
-          ((unsigned long) &(c.c)) - ((unsigned long) &(c.a)));
+          (reinterpret_cast<unsigned long>(&(c.c)) - reinterpret_cast<unsigned long>(&(c.a))));
   EXPECT_EQ(Cpp::GetVariableOffset(datamembers[3]),
-          ((unsigned long) &(c.d)) - ((unsigned long) &(c.a)));
-
+          (reinterpret_cast<unsigned long>(&(c.d)) - reinterpret_cast<unsigned long>(&(c.a))));
+#endif // WIN32
```

Comment:
These do not seem right.

Reply:
This is something I'm planning to revert. It was from when I was trying to debug the Windows compile issue.

Reply:
I tried many times to geet this part of the test to work on Windows without sucess.

Reply:
I've reverted this change now.